sdks/python/apache_beam/io/external/xlang_jdbcio_it_test.py

```diff
@@ -179,8 +180,8 @@ def test_xlang_jdbc_read(self, database):
     self._setUpTestCase(container_init, db_string, driver)
     table_name = 'jdbc_external_test_read'
     self.engine.execute(
-        "CREATE TABLE {}(f_int INTEGER, f_timestamp TIMESTAMP)".format(
-            table_name))
+        "CREATE TABLE {}(f_int INTEGER, f_timestamp TIMESTAMP, f_decimal DECIMAL(10,2))"  # pylint: disable=line-too-long
+        .format(table_name))
```

Comment:
Can you also add a test(s) for this in standard_coders.yaml?

Reply:
added in standard_coders.yaml